packages/framer-motion/src/animation/legacy-popmotion/index.ts

```diff
@@ -124,7 +127,7 @@ export function animate<V = number>({
             isComplete = isForwardPlayback ? state.done : elapsed <= 0
         }
 
-        onUpdate?.(latest)
+        onUpdate && onUpdate(latest)
```

Comment:
A slight aside to this PR but replacing `?.` for `foo &&` makes a surprising saving for transpilation.

Reply:
🤔 Why?

Reply:
It's just super-exact. `onUpdate?.()` becomes:

```
var _onUpdate;
(_onUpdate = onUpdate) === null || _onUpdate === void 0 ? void 0 : _onUpdate();
```

`??` is also upsettingly large. `const a = 0 ?? 1`:

```javascript
const a = (_ = 0) !== null && _ !== void 0 ? _ : 1;
```